app/src/main/java/nyc/c4q/jonathancolon/inContaq/contactlist/PicassoHelper.java

```diff
@@ -13,11 +13,16 @@ public PicassoHelper(Context context) {
         this.context = context;
     }
 
+    public void preloadImages(String string){
+        Picasso.with(context).load(string).fetch();
+    }
+
     public void loadImageFromUri(Uri uri, ImageView imageView){
-        Picasso.with(context).load(uri).into(imageView);
+        Picasso.with(context).load(uri).resize(1000, 1000).centerInside().into(imageView);
     }
 
     public void loadImageFromString(String uriString, ImageView imageView){
```

Comment:
They do the exact same thing, except loadImageFromString() just converts the String to a Uri using Uri.parse(uriString). So you could just do

```java
loadImageFromString(String uriString, ImageView imageView) {
    loadImageFromUri(Uri.parse(uriString), imageView);
}
```